fitbenchmarking/controllers/bumps_controller.py

```diff
@@ -28,17 +28,17 @@ class BumpsController(Controller):
                     'scipy-leastsq',
                     'dream'],
             'ls': ['lm-bumps', 'scipy-leastsq'],
-            'deriv_free': ['amoeba', 'de', 'dream'],
-            'general': ['amoeba', 'newton', 'de', 'dream'],
+            'deriv_free': ['amoeba', 'de'],
+            'general': ['amoeba', 'newton', 'de'],
             'simplex': ['amoeba'],
             'trust_region': ['lm-bumps', 'scipy-leastsq'],
             'levenberg-marquardt': ['lm-bumps', 'scipy-leastsq'],
             'gauss_newton': [],
             'bfgs': ['newton'],
             'conjugate_gradient': [],
             'steepest_descent': [],
-            'global_optimization': ['de', 'dream'],
```

Comment:
I assume dream was under global as its considered to be like a swarm technique? Do we want to include it still in the old categories as that is how it is used?

Reply:
I'd removed dream from the other algorithm type categories as the MCMC algorithm type is incompatible with all other algorithm types - we could run dream differently (reporting chi squared values rather than confidence values for accuracy) depending on the algorithm type chosen by the user but I think this would get confusing

Reply:
that makes sense to me. I dont like reducing bayes to a parameter + gaussian error